src/volt/components/Menu/index.js

```diff
@@ -8,11 +8,12 @@ const Menu = ({ onClose, account }) => (
     <Container>
       <DeoraLogo />
       <MenuItems>
-        <Item href="/page-1">Über uns</Item>
-        <Item href="/page-2">Quadratical Voting</Item>
-        <Item href="/page-3">Die Blockchain</Item>
-        <Item href="/page-4">Sicherheit & Privacy</Item>
-        <Item href="/page-5">Wahlergebnisse</Item>
+        <Item target="_blank" href="https://www.deora.earth/volt-platform/about">Über uns</Item>
```

Comment:
All those links give 404, will they be available later?

Reply:
Last one will be available once the event is over